### PR TITLE
V0.8.1

### PR DESCRIPTION
**Pull request by izahn**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes https://github.com/conda-forge/jupyterlab-sos-feedstock/pull/16

### CODE REVIEW

**Comment by BoPeng** [comment]
@izahn Thanks for your PR but I do not understand why `pip check` fails after the installation of `jupyterlab-transient-display-data`.

**Comment by izahn** [comment]
Yes, interesting that we didn't have this problem over in https://github.com/conda-forge/jupyterlab-sos-feedstock/pull/15/checks

The reason I changed the test from
```
jupyter labextension list | grep jupyterlab-sos  # [unix]
```
to 
```
pip check
```
is that `jupyter labextension list` doesn't write to standard out, so we can't pipe and grep (ugh, why!?). I'll go back to a version of the old test based on https://github.com/conda-forge/jupyter-lsp-feedstock/blob/master/recipe/meta.yaml#L56 and see if that works.

**Comment by izahn** [comment]
I think this is OK now and can be merged. Also happy to add myself as a co-maintainer if you like.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,12 +27,14 @@ requirements:
     - jupyterlab-transient-display-data
     - nodejs
     - python >=3.6
+    - sos
 
 test:
   commands:
-    - pip check
```
Comment (BoPeng) [inline recipe/meta.yaml:33]: so we still do not know what went wrong with pip check.

**Review by BoPeng — APPROVED** [review]

**Comment by BoPeng** [comment]
> I think this is OK now and can be merged. Also happy to add myself as a co-maintainer if you like.

Just added you as a maintainer and merged the PR. Thanks.

**Pull request merged**